node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
+  "systemParams": "darwin-x64-83",
+  "modulesFolders": [],
+  "flags": [],
+  "linkedModules": [],
+  "topLevelPatterns": [],
+  "lockfileEntries": {},
+  "files": [],
+  "artifacts": {}
+}
```

Comment:
Remove this file please

Reply:
I have not been able to find this one to delete it. I see where it is from that bit of the path but when I search for it the only place I see it is in my .gitignore. If you've got any idea on how to help find that to delete it please let me know.